### DISCUSSION TITLE
Guest Page

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#8632] [General]
How can I create a page for a pre regristration outside the admin panel? (Guest users) I know I need to create a livewire component, I have the livewire page and the view, but when i add the route, is not working, it says missing layout or $this is not in context when i try to create a form to load the data, any examples?

**Comment by krekas** [comment]
To me it seems like you need to check how Livewire works. Otherwise your problem needs more context. Creating pages outside of panel shouldn't have anything in common with the panel

**Reply by ramiropiro** [reply]
I just want to know, for example, how to make a registration form, thats outside the panel, no need to login, with filament forms

**Reply by krekas** [reply]
Well do you know how laravel and livewire works first? Because it seams do don't know. So first you should learn them.
But for your questions there is a docs section which you haven't tried to read seems like it https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component

**Reply by ramiropiro** [reply]
thanks man, very helpfull, Im gonna learn again laravel and liviwire, you are doin a great job in this community

**Comment by wychoong** [comment]
since you want to show the page outside of admin panel, you should register your routes/pages in web.php
for the missing layout, read livewire doc
for $this, thats php that you need to understand

show code or no one can help by guessing
